templates/app/assets/javascripts/solidus_starter_frontend.js

```diff
@@ -1,10 +1,11 @@
 // Placeholder manifest file.
 // the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/frontend/all.js'
 
-// This is generated by `solidus:install` at
-// `vendor/assets/javascripts/spree/frontend/all.js`. See `setup_assets` at
+// spree/frontend/all points to the
+// `vendor/assets/javascripts/spree/frontend/all.js` file generated by
+// `solidus:install`. See `setup_assets` at
 // https://github.com/solidusio/solidus/blob/master/core/lib/generators/solidus/install/install_generator.rb
-//= require spree/frontend/all
+// = require spree/frontend/all
```

Comment:
Not sure about this, but does the space stops it from being loaded or is just fo alignment?

Reply:
Thanks for the catch @elia ! I'll remove the space.